constance/utils.py

```diff
@@ -20,3 +20,27 @@ def get_values():
     default_initial = ((name, options[0]) for name, options in settings.CONFIG.items())
     # Then update the mapping with actually values from the backend
     return dict(default_initial, **dict(config._backend.mget(settings.CONFIG)))
+
+def get_values_for_keys(keys):
```

Comment:
```suggestion
def get_values_for_keys(keys: list):
```

Reply:
This you restrict the type to `list`. Should I check only for lists?

While writing, I've thought one case that maybe worth testing: pass the same value multiple times to the list and see what happens. What do you think?

Maybe expect `set` only is a better restriction?

Reply:
Enforcing a `set` would avoid the library to check duplicated keys as it can happen in a `list`.

Reply:
Iterable[str] looks like the right typing here